driver/src/contracts/stablex_contract.rs

```diff
@@ -139,7 +140,11 @@ impl StableXContract for BatchExchange {
             prices,
             token_ids_for_price,
         )
-        .gas_price(GasPrice::Scaled(3.0))
+        .gas_price(
+            gnosis_safe_gas_station::get_gas_price()
+                .map(|gas_price| GasPrice::Value(gas_price.fast))
+                .unwrap_or(GasPrice::Scaled(3.0)),
```

Comment:
For fast I think we don't need to scale it with a factor 3. Let's start maybe with 1.5?

Reply:
It doesn't scale `gas_price.fast` by 3. If it cannot retrieve the gas price from the api then it uses ethcontract's built in `GasPrice::Scaled` which is the same behavior as before.
Or are you saying the old behavior should have been 1.5 all along?

Reply:
Yes, my bad. that's good